core/zoom-controls.js

```diff
@@ -0,0 +1,245 @@
+/**
+ * @license
+ * Visual Blocks Editor
+ *
+ * Copyright 2011 Google Inc.
+ * https://developers.google.com/blockly/
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Object representing a zoom icons.
+ * @author carloslfu@gmail.com (Carlos Galarza)
+ */
+'use strict';
+
+goog.provide('Blockly.ZoomControls');
+
+goog.require('goog.dom');
+
+
+/**
+ * Class for a zoom controls.
+ * @param {!Blockly.Workspace} workspace The workspace to sit in.
+ * @constructor
+ */
+Blockly.ZoomControls = function(workspace) {
+  this.workspace_ = workspace;
+};
+
+/**
+ * URL of the zoom-in image.
+ * @type {string}
+ * @private
+ */
+Blockly.ZoomControls.prototype.ZOOMIN_URL_ = 'zoomin.svg';
```

Comment:
// TODO: Add these images to sprites.png so that we don't have three separate http calls.

This can be done later, separate images are fine for now.